src/App/App.tsx

```diff
@@ -1,42 +1,119 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import USAState from '../USAState/USAState';
 import ComparisonContainer from '../ComparisonContainer/ComparisonContainer';
 import Stat from '../Stat/Stat';
+import { fetchAllCurrentUSAData, fetchCurrentStateData } from '../apiCalls'
+import { GiCoffin } from 'react-icons/gi';
+import { RiVirusFill, RiHospitalFill } from 'react-icons/ri';
 import './App.scss';
 
 interface Props { 
 }
 
 interface State {
-  selectedUSAState: {};
+  allUSAData: Details, 
+  selectedUSAState: Details
 }
 
-class App extends Component<Props,State> {
+interface Details {
```

Comment:
Will need to circle back to the use of optional parameters to ensure they are being used and passed correctly.